contrib/externs/angular-1.5.js

```diff
@@ -2468,3 +2467,41 @@ angular.$timeout_.cancel = function(promise) {};
 
 /** @typedef {!Window} */
 angular.$window;
+
+/******************************************************************************
+ * $locale Service
+ *****************************************************************************/
+
+ /**
+ * @typedef {{
+ *   DATETIME_FORMATS: {
```

Comment:
I think a better question is do we want to encourage their use by including it in official externs?